translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"source $(brew --prefix)/etc/bash_completion\n"
+"\" >> $HOME/.bash_profile\n"
+"\t\tsource $HOME/.bash_profile\n"
+"\n"
+"\t\t# Load the kubectl completion code for bash into the current shell\n"
+"\t\tsource <(kubectl completion bash)\n"
+"\n"
+"\t\t# Write bash completion code to a file and source if from .bash_profile\n"
+"\t\tkubectl completion bash > ~/.kube/completion.bash.inc\n"
+"\t\tprintf \"\n"
+"# Kubectl shell completion\n"
+"source '$HOME/.kube/completion.bash.inc'\n"
+"\" >> $HOME/.bash_profile\n"
+"\t\tsource $HOME/.bash_profile\n"
+"\n"
+"\t\t# Load the kubectl completion code for zsh[1] into the current shell\n"
+"\t\tsource <(kubectl completion zsh)"
+msgstr ""
+"\n"
+"\t\t# Instala o bash completion no Mac utilizando homebrew\n"
+"\t\tbrew install bash-completion\n"
+"\t\tprintf “\n"
+"# Bash completion support\n"
+"source $(brew —prefix)/etc/bash_completion\n"
+"“ >> $HOME/.bash_profile\n"
+"\t\tsource $HOME/.bash_profile\n"
+"\n"
+"\t\t# Carrega o kubectl completion code para o bash no shell corrent\n"
+"\t\tsource <(kubectl completion bash)\n"
+"\n"
+"\t\t# Escreve o bash completion code em um arquivo e faz o source se é para "
+"o .bash_profile\n"
+"\t\tkubectl completion bash > ~/.kube/completion.bash.inc\n"
+"\t\tprintf “\n"
+"# Kubectl shell completion\n"
+"source ‘$HOME/.kube/completion.bash.inc’\n"
+"“ >> $HOME/.bash_profile\n"
+"\t\tsource $HOME/.bash_profile\n"
+"\n"
+"\t\t# Carrega o kubectl completion code para zsh[1] no shell corrent\n"
+"\t\tsource <(kubectl completion zsh)"
+
+#: pkg/kubectl/cmd/get.go:64
+msgid ""
+"\n"
+"\t\t# List all pods in ps output format.\n"
+"\t\tkubectl get pods\n"
+"\n"
+"\t\t# List all pods in ps output format with more information (such as node "
+"name).\n"
+"\t\tkubectl get pods -o wide\n"
+"\n"
+"\t\t# List a single replication controller with specified NAME in ps output "
+"format.\n"
+"\t\tkubectl get replicationcontroller web\n"
+"\n"
+"\t\t# List a single pod in JSON output format.\n"
+"\t\tkubectl get -o json pod web-pod-13je7\n"
+"\n"
+"\t\t# List a pod identified by type and name specified in \"pod.yaml\" in "
+"JSON output format.\n"
+"\t\tkubectl get -f pod.yaml -o json\n"
+"\n"
+"\t\t# Return only the phase value of the specified pod.\n"
+"\t\tkubectl get -o template pod/web-pod-13je7 --template={{.status.phase}}\n"
+"\n"
+"\t\t# List all replication controllers and services together in ps output "
+"format.\n"
+"\t\tkubectl get rc,services\n"
+"\n"
+"\t\t# List one or more resources by their type and names.\n"
+"\t\tkubectl get rc/web service/frontend pods/web-pod-13je7\n"
+"\n"
+"\t\t# List all resources with different types.\n"
+"\t\tkubectl get all"
+msgstr ""
+"\n"
+"\t\t# Lista todos os pods no formato de saída ps.\n"
+"\t\tkubectl get pods\n"
+"\n"
+"\t\t# Lista todos os pods no formato de saída ps com mais informações (como "
+"o nome do node).\n"
+"\t\tkubectl get pods -o wide\n"
+"\n"
+"\t\t# Lista um único replication controller com o NAME especificado no "
```

Comment:
I'm thinking here if we actually want to translate the "NAME" or if we can assume this is the field name and leave as "NAME"